myuw/templates/handlebars/card/schedule/visual.html

```diff
@@ -4,155 +4,159 @@
-	        <div style="font-size:.8em; color:#555; text-align:center; margin-top:3px;">
-                {{#if building_tbd}}
-                    Room TBD
-                {{else}}
-                    {{#if latitude}}
-                        <a href="http://maps.google.com/maps?q={{ latitude }},{{ longitude }}+({{encodeForMaps building_name }})&z=18" class="show_visual_map" rel="{{building_name}}">
+        {{#if courses_meeting_tbd }}
+        <div class="courseTBD" aria-hidden="true">
+            {{#each courses_meeting_tbd }}
+    	    <div class="pull-left course-box">
+    	        <div class="c{{color_id}} course-info"><a href="/{{#if ../is_instructor}}teaching{{else}}academics{{/if}}/#{{curriculum}}-{{course_number}}-{{section_id}}" style="color: white;" class="show_section_card">{{curriculum}} {{course_number}}&nbsp;{{section_id}}</a></div>
+    	        <div style="font-size:.8em; color:#555; text-align:center; margin-top:3px;">Room TBD</div>
+    	    </div>
+            {{/each}}
+        </div>
+        {{/if}}
+        <!-- end - if there are meetings with times to be arranged -->
+    
+            <!-- begin - if there are meetings with times to be arranged -->
+        {{#if courses_no_meeting }}
+        <div class="courseTBD" aria-hidden="true">
+            <p>Following courses have not specified a meeting time:</p>
+            {{#each courses_no_meeting }}
+    	    <div class="pull-left course-box">
+    	        <div class="c{{color_id}} course-info"><a href="/{{#if ../is_instructor}}teaching{{else}}academics{{/if}}/#{{curriculum}}-{{course_number}}-{{section_id}}" style="color: white;" class="show_section_card">{{curriculum}} {{course_number}}&nbsp;{{section_id}}</a></div>
+    	        <div style="font-size:.8em; color:#555; text-align:center; margin-top:3px;">
+                    {{#if building_tbd}}
+                        Room TBD
+                    {{else}}
+                        {{#if latitude}}
+                            <a href="http://maps.google.com/maps?q={{ latitude }},{{ longitude }}+({{encodeForMaps building_name }})&z=18" class="show_visual_map" rel="{{building_name}}">
+                        {{/if}}
+                        {{building}} {{room}}
+                        {{#if latitude}}</a>{{/if}}
                     {{/if}}
-                    {{building}} {{room}}
-                    {{#if latitude}}</a>{{/if}}
-                {{/if}}
-            </div>
-	    </div>
-        {{/each}}
+                </div>
+    	    </div>
+            {{/each}}
+        </div>
+        {{/if}}
+    </div>
+        <!-- end - if there are meetings with times to be arranged -->
+        
+        
+        <!--Adding SR only final exam schedule-->
```

Comment:
Since this is a temporary fix until Stephen is done with his work, why not just comment out this entire table? Seems weird to me to keep a screen-reader only table and content when the table is still empty. 

Might be better to hide the entire card from screen readers until it can get fixed properly. At this point the only content visible to screen readers is a few headings.

Reply:
yeah, I thought about that option too.